src/renderer/containers/App/handleUploadJobUpdates.ts

```diff
@@ -98,15 +98,15 @@ export function handleUploadJobUpdates(job: JSSJob, dispatch: Dispatch<any>) {
       job.currentStage === UploadStatus.RETRY
     ) {
       dispatch(receiveFSSJobCompletionUpdate(fssJob));
-    }
-
-    // Otherwise, report progress
-    // TODO Multifile FSS jobs are currently identifiable by their service_fields.subfiles property,
-    //       but more ideally would just be labeled with something like "multifile: true".
-    if (fssJob.serviceFields?.subfiles) {
-      handleFSSMultifileJobUpdate(fssJob, dispatch);
     } else {
-      handleFSSJobUpdate(fssJob, dispatch);
+        // Otherwise, report progress
```

Comment:
Weird diff - it looks like the change is only that the block got indented?

Reply:
Yeah the diff makes it hard to read. The change is that everything following line 100 is now in a separate `else`. So if the `jobs.serviceFields.fileId || ...` condition starting on line 96 is met, the rest of the code gets skipped.